whois-nrtm4/src/test/java/net/ripe/db/nrtm4/NrtmProcessorIntegrationTest.java

```diff
@@ -0,0 +1,67 @@
+package net.ripe.db.nrtm4;
+
+import net.ripe.db.nrtm4.persist.DeltaFileModel;
+import net.ripe.db.nrtm4.persist.NrtmSourceHolder;
+import net.ripe.db.nrtm4.persist.NrtmVersionInfoRepository;
+import net.ripe.db.nrtm4.persist.RpslObjectDao;
+import net.ripe.db.whois.common.dao.jdbc.AbstractDatabaseHelperIntegrationTest;
+import net.ripe.db.whois.common.domain.Timestamp;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.Tag;
+import org.junit.jupiter.api.Test;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.test.context.ContextConfiguration;
+
+import static net.ripe.db.whois.common.dao.jdbc.JdbcRpslObjectOperations.loadScripts;
+import static net.ripe.db.whois.common.dao.jdbc.JdbcRpslObjectOperations.truncateTables;
+import static org.hamcrest.MatcherAssert.assertThat;
+import static org.hamcrest.Matchers.is;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+
+@Tag("IntegrationTest")
+@ContextConfiguration(locations = {"classpath:applicationContext-nrtm4-test.xml"})
+public class NrtmProcessorIntegrationTest extends AbstractDatabaseHelperIntegrationTest {
+
+    @Autowired
+    private RpslObjectDao rpslObjectDao;
```

Comment:
I think that you are not using this @Autowired

Reply:
fixed